content/en/tutorials/importing-first-model-and-animation.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Importing your first Model and Animation
+layout: tutorial-page.hbs
+tags: animation,basics
+thumb: /images/tutorials/importing-first-model-and-animation/thumbnail.jpg
+---
+
+<iframe src="https://www.youtube.com/embed/r0LYQw7laRA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Overview
+
+In this tutorial, we will be taking you through adding your first animated model to your project as you can see here.
+
+![][preview-image]
+
+## Getting our example assets
+
+For this, we will be using Kenney's Animated Characters assets pack which can be downloaded from [their site][kenney-site].
+
+![][kenney-site-image]
+
+After downloading, unzip the file and let's have a look at the contents.
+
+They've split the model and animation files from each other which is perfect for us.
+
+![][asset-pack-contents-image]
+
+## Importing the model FBX
+
+We are interested in the model file to begin with.
+
+Let's open the model folder and drag the FBX file into our project assets panel.
+
+This will import the file into our project and PlayCanvas Editor will create materials, container, render assets, and most importantly a template asset.
```

Comment:
```suggestion
This will import the file into our project and the PlayCanvas Editor will create materials, container, render assets and, most importantly, a template asset.
```